setup.py

```diff
@@ -2,13 +2,7 @@
 import os
```

Comment:
Given that we're no longer installing ichnaea Python package, we might not need this `setup.py` file anymore.